TestControllers/TestController.psm1

```diff
@@ -139,6 +139,13 @@ Class TestController
 
 		$parameterErrors = @()
 		# Validate general parameters
+		if (!$this.RGIdentifier) {
```

Comment:
RDIdentifier is not a must for other platforms

Reply:
updated, thanks Liz!